ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-ref.yaml

```diff
@@ -0,0 +1,43 @@
+ ref:
+  as: cucushift-logging-e2e
+  from_image:
+    namespace: ci
+    name: verification-tests
+    tag: latest
+  grace_period: 10m
+  timeout: 16h
+  commands: cucushift-logging-e2e-commands.sh
+  cli: latest
+  resources:
+    limits:
+      ci-operator.openshift.io/shm: 2G
+      cpu: "2"
+      memory: 6Gi
+    requests:
+      ci-operator.openshift.io/shm: 2G
+      cpu: 800m
+      memory: 1Gi
+  env:
+  - name: E2E_RUN_TAGS
+    default: "@aws-ipi"
+    documentation: "Executed cucumber tags"
+  - name: E2E_SKIP_TAGS
+    default: "not @customer
+          and not @destructive
```

Comment:
Most of the logging cases have `@destructive` tag, so I think we should not add this tag here.